src/main/java/seedu/bankwithus/Account.java

```diff
@@ -16,6 +18,15 @@ public class Account {
     public Account(String name, String balance) {
         this.name = name;
         this.balance = balance;
+        this.withdrawalChecker = new WithdrawalChecker();
+    }
+
+    //@@author tyuyang
+    public Account(String name, String balance, String totalAmtWithdrawn,
+            LocalDate lastWithdrawnDate) {
+        this.name = name;
+        this.balance = balance;
+        this.withdrawalChecker = new WithdrawalChecker(totalAmtWithdrawn, lastWithdrawnDate);
```

Comment:
Created new constructor and modified old constructor to instantiate withdrawalChecker